### ISSUE TITLE
关于L_ranknet

### ISSUE THREAD

**Issue by DryPilgrim** [sunnweiwei/RankGPT#28]
根据论文公式L_ranknet, ri < rj时：
若si-sj = 0.4, loss = 0.908
若si-sj = -0.4, loss = 0.511

出现矛盾：我们希望si-sj 越大，loss越小， 但现实是si-sj 越小，loss越小。

<img width="458" alt="image" src="https://github.com/user-attachments/assets/5db6a147-6f94-4201-b012-9fef4224e0cd" />

**Comment by sunnweiwei** [comment]
Hi, Thank you for pointing out this issue!

The correct formula should be:
![image](https://github.com/user-attachments/assets/ca05c290-1ad0-436a-9f92-33b9341767f0)

I have submitted an update to arXiv to fix it.
